src/games/strategy/util/SoftJEditorPane.java

```diff
@@ -10,47 +10,47 @@
  * Example, when a JEditorPane has rich HTML in it, with huge images.
  */
 public class SoftJEditorPane {
-  protected SoftReference<JEditorPane> m_component;
-  protected final String m_text;
+  protected SoftReference<JEditorPane> component;
+  protected final String text;
 
   public SoftJEditorPane(final String text) {
-    m_text = text;
+    this.text = text;
   }
 
   protected JEditorPane createComponent() {
     final JEditorPane pane = new JEditorPane();
     pane.setEditable(false);
     pane.setContentType("text/html");
-    pane.setText(m_text);
+    pane.setText(text);
     pane.setCaretPosition(0);
     return pane;
   }
 
   public synchronized JEditorPane getComponent() {
-    if (m_component == null) {
-      m_component = new SoftReference<JEditorPane>(createComponent());
+    if (component == null) {
+      component = new SoftReference<JEditorPane>(createComponent());
     }
-    JEditorPane component = m_component.get();
+    JEditorPane component = this.component.get();
```

Comment:
Can we rename the local variable so we don't have a local and member variable with the same name?